tools/tokamak_grids/gridgen/create_nonorthogonal.pro

```diff
@@ -1080,13 +1146,13 @@ FUNCTION create_nonorthogonal, F, R, Z, in_settings, critical=critical, $
     start_ri = (SMOOTH([start_ri[(np-s):(np-1)], start_ri, start_ri[0:(s-1)]], s))[s:(np-1+s)]
     start_zi = (SMOOTH([start_zi[(np-s):(np-1)], start_zi, start_zi[0:(s-1)]], s))[s:(np-1+s)]
     
-    FOR i=0, np-1 DO BEGIN
-      follow_gradient_nonorth, interp_data, R, Z, start_ri[i], start_zi[i], start_f, ri1, zi1
-      start_ri[i] = ri1
-      start_zi[i] = zi1
-    ENDFOR
+    ;FOR i=0, np-1 DO BEGIN
```

Comment:
Is this refinement not needed, or does it lead to incorrect results?

Reply:
I thought it was not needed, because I switched on refinement in `grid_region_nonorth`. Looking again, they're not quite the same thing, because the refinement I switched on is of the line after interpolating onto the poloidal grid, whereas this was for the line that is input to the interpolation. I guess more refinement should never hurt and might help, so I'll put back this stage of refinement (possibly at the beginning of `grid_region_nonorth` if that reduces code duplication.